src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -152,7 +143,7 @@ const EnvironmentPicker = ({
     useEffect(() => {
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186236413-54586af5-2204-4319-ac65-622a3dc5f8e1.png)

Reply:
had commented and marked as "resolved" this on the original PR